Samples/Notifications/Push/cpp-console-unpackaged/cpp-console-unpackaged.cpp

```diff
@@ -0,0 +1,160 @@
+                LOG_HR_MSG(
+                    args.extendedError,
+                    "The WNS Channel URI request is in back-off retry mode because of a retryable error! Expect delays in acquiring it. RetryCount = %d",
+                    args.retryCount);
+            }
+        });
+
+    auto result{ co_await channelOperation };
+
+    if (result.Status() == PushNotificationChannelStatus::CompletedSuccess)
+    {
+        auto channel{ result.Channel() };
+
+        std::cout << "\nWNS Channel URI: " << winrt::to_string(channel.Uri().ToString()) << std::endl;
+
+        // It's the caller's responsibility to keep the channel alive
+        co_return channel;
+    }
+    else if (result.Status() == PushNotificationChannelStatus::CompletedFailure)
+    {
+        LOG_HR_MSG(result.ExtendedError(), "We hit a critical non-retryable error with the WNS Channel URI request!");
+        co_return nullptr;
+    }
+    else
+    {
+        LOG_HR_MSG(result.ExtendedError(), "Some other failure occurred.");
+        co_return nullptr;
+    }
+
+};
+
+winrt::Microsoft::Windows::PushNotifications::PushNotificationChannel RequestChannel()
+{
+    auto task{ RequestChannelAsync() };
+    if (task.wait_for(std::chrono::minutes(5)) != AsyncStatus::Completed)
+    {
+        task.Cancel();
+        return nullptr;
+    }
+
+    auto result{ task.GetResults() };
+    return result;
+}
+
+// Subscribe to an event which will get signaled whenever a foreground notification arrives.
+void SubscribeForegroundEventHandler()
+{
+    winrt::event_token token{ PushNotificationManager::Default().PushReceived([](auto const&, PushNotificationReceivedEventArgs const& args)
+    {
+        auto payload{ args.Payload() };
+
+        std::string payloadString(payload.begin(), payload.end());
+        std::cout << "\nPush notification content received in the FOREGROUND: " << payloadString << std::endl;
+    }) };
+}
+
+int main()
+{
+    // Setup an event handler, so we can receive notifications in the foreground while the app is running.
+    SubscribeForegroundEventHandler();
+
+    PushNotificationManager::Default().Register();
+
+    auto args{ AppInstance::GetCurrent().GetActivatedEventArgs() };
+    switch (args.Kind())
+    {
+        // When it is launched normally (by the users, or from the debugger), the sample requests a WNS Channel URI and
+        // displays it, then waits for notifications. This user can take a copy of the WNS Channel URI and use it to send
+        // notifications to the sample
+        case ExtendedActivationKind::Launch:
+        {
+            // Request a WNS Channel URI which can be passed off to an external app to send notifications to.
+            // The WNS Channel URI uniquely identifies this app for this user and device.
+            PushNotificationChannel channel{ RequestChannel() };
+            if (!channel)
+            {
+                std::cout << "\nThere was an error obtaining the WNS Channel URI" << std::endl;
```

Comment:
Same comment with \n here. IF the prior console output uses endl, this isn't really needed.

Reply:
front_end/panels/animation/animationTimeline.css
front_end/panels/application/components/trustTokensViewDeleteButton.css
front_end/panels/console/consolePinPane.css
front_end/panels/css_overview/cssOverviewCompletedView.css
front_end/panels/elements/components/adornerSettingsPane.css
front_end/panels/elements/components/computedStyleTrace.css
front_end/panels/elements/components/elementsBreadcrumbs.css
front_end/panels/elements/computedStyleWidgetTree.css
front_end/panels/elements/elementsTreeOutline.css
front_end/panels/emulation/deviceModeView.css
front_end/panels/event_listeners/eventListenersView.css
front_end/panels/issues/components/hideIssuesMenu.css
front_end/panels/media/playerListView.css
front_end/panels/network/networkLogView.css
front_end/panels/network/requestPayloadTree.css
front_end/panels/network/signedExchangeInfoTree.css
front_end/panels/settings/emulation/components/userAgentClientHintsForm.css
front_end/panels/settings/emulation/devicesSettingsTab.css
front_end/panels/sources/watchExpressionsSidebarPane.css
front_end/panels/webauthn/webauthnPane.css
front_end/third_party/codemirror/package/addon/fold/foldgutter.css
front_end/ui/components/adorners/adorner.css
front_end/ui/components/buttons/button.css
front_end/ui/components/data_grid/dataGrid.css
front_end/ui/components/icon_button/iconButton.css
front_end/ui/legacy/TabbedPane.ts
front_end/ui/legacy/tabbedPane.css
front_end/ui/legacy/textButton.css
front_end/ui/legacy/toolbar.css